snmp/datadog_checks/snmp/utils.py

```diff
@@ -287,7 +288,7 @@ def oid_dict(self, key, value):
             displayed = values[0]
         else:
             displayed = '{{{}}}'.format(', '.join(values))
-        return "'{}': {}".format(key, displayed)
+        return "'{}': {}".format(key, re.sub('\x00', '', displayed))
```

Comment:
Any reason to use re.sub? It's way more expensive than a regular replace.

Reply:
I was under the impression that replace was a python3 only thing; apparently not. will adjust